src/box/sql/parse.y

```diff
@@ -289,7 +289,7 @@ carglist ::= .
 cconsname(N) ::= CONSTRAINT nm(X). { N = X; }
 cconsname(N) ::= . { N = Token_nil; }
 ccons ::= DEFAULT term(X).            {sql_add_term_default(pParse, &X);}
-ccons ::= DEFAULT LP expr(X) RP.      {sqlAddDefaultValue(pParse,&X);}
+ccons ::= DEFAULT LP expr(X) RP.      {sql_add_func_default(pParse, &X);}
```

Comment:
This is weird. These 2 examples produce different defaults: the first one makes a default constant, the second one - a default function:
```Lua
box.cfg{}
box.execute('CREATE TABLE test (id INT PRIMARY KEY, val INT DEFAULT 100)')
box.execute('CREATE TABLE test2 (id INT PRIMARY KEY, val INT DEFAULT (100))')
```
Look at their `:format()` results to see the difference in the `val` column. Can you do something with it and not depend on the parentheses so much?

Reply:
@ImeevMA, what about making identifier a mandatory token for this syntax?
E.g. something like `ccons ::= DEFAULT ID LP expr(X) RP`, if I didn't mess in lemon notation.

Reply:
I actually thought that it is quite good rule - if in parentheses than a function, otherwise a term. But you are right, this rule does sound strange for users, probably. Fixed.